appveyor.yml

```diff
@@ -11,48 +11,47 @@ environment:
     global:
         # MSVC Express 2008's setenv.cmd failes if /E:ON and /V:ON are not
         # enabled in the batch script interpreter
-        #
-        #      WITH_COMPILER: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_compiler.cmd"
         CMD_IN_ENV: cmd /E:ON /V:ON /C .\appveyor\run_with_env.cmd
 
     matrix:
         # For Python versions available on Appveyor, see
-        # http://www.appveyor.com/docs/installed-software#python
-      - {PYVER: "27", PYTHON_ARCH: "32"}
-      - {PYVER: "27", PYTHON_ARCH: "64"}
-      - {PYVER: "34", PYTHON_ARCH: "32"}
-      - {PYVER: "34", PYTHON_ARCH: "64"}
-      - {PYVER: "35", PYTHON_ARCH: "32"}
-      - {PYVER: "35", PYTHON_ARCH: "64"}
-      - {PYVER: "36", PYTHON_ARCH: "32"}
-      - {PYVER: "36", PYTHON_ARCH: "64"}
-      - {PYVER: "37", PYTHON_ARCH: "32"}
-      - {PYVER: "37", PYTHON_ARCH: "64"}
-
+        # https://www.appveyor.com/docs/build-environment/
```

Comment:
My only concern with removing all the matrix options is we might not be able to build 'fresh' all the C libraries and do the testing for the different versions before the VM 'resets'.  From what I remember, Appveyor has a maximum build time of 60 minute, but each matrix item is considered a new build and resets the counter.  Tho I might be being a little over cautious, checking out the last builds, the total time was less than 6 minutes each, so for the 10 different builds we should be OK.   Another thing in our favor is that we really only have 6 versions of VC to build, so we reshare the openssl and libpq libraries.  Something to keep in mind, tho, if adding another Python version in the future doesn't complete the build.

Another interesting tidbit that doesn't apply in this situation: I didn't realize when I first started with the Appveyor build that each matrix item also has a separate cache, so my attempt at separating out the libraries based upon VC versions and architect builds wasn't necessary.  Of course, it makes sense with a combined build like this to separate out the libraries based upon version and architect.